src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
+
+  static newAnimal(animal) {
+    Animal.alive.push(animal);
+  }
```

Comment:
Actually, this static method is unnecessary. Just push into `Animal.alive` array, where it's needed